rolling/distribution.yaml

```diff
@@ -1500,6 +1500,13 @@ repositories:
       url: https://github.com/ros-drivers/phidgets_drivers.git
       version: galactic
     status: maintained
+  plotjuggler_msgs:
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://github.com/facontidavide/plotjuggler_msgs-release.git
+      version: 0.1.2-1
+    status: developed
```

Comment:
@facontidavide no `doc` nor `source` entries?

Reply:
@facontidavide Friendly ping. Please add a `source` entry (and optionally a `doc` entry).